src/public/js/components/Breadcrumb.js

```diff
@@ -3,10 +3,12 @@ import React from 'react';
 /*
 interface BreadcrumbProps{
     breadcrumb: string[], // array of content identifiers
+    textsById: Map<ContentId, FinanceElementTextsRecord>
     onContentChange: (string) => void // (side effect), displays another page
 }
  */
-export default function ({breadcrumb, onContentChange}) {
+
+export default function ({breadcrumb, textsById, onContentChange}) {
     const children = [];
 
     breadcrumb.forEach((e, i) => {
```

Comment:
Eye candy: wouldn't be nicer to split that in three operations? Like:

1. `const last = breadcrumb.pop()`
2. `const children = breadcrumb.map`
3.  `children.push(last)`

It does not change the logic but get rid of the if/else and makes explicit the case of the last element without having to write any comment about it.